src/templates/template-common/README.md

```diff
@@ -68,7 +68,11 @@ torchrun \
   --node_rank 0 \
   --master_addr #:::= it.master_addr :::# \
   --master_port #:::= it.master_port :::# \
-  main.py config.yaml --backend #:::= it.backend :::#
+  main.py config.yaml --backend #:::= it.backend :::# \
```

Comment:
Ideally, we would like to add links to config system docs in the README. Maybe, before `## Training` section ?

Reply:
@vfdev-5 I was planing to add something like this in the end 

Note: We use Python-Fire as the default argument parser here. For more information refer the [docs](https://github.com/google/python-fire/blob/master/docs/guide.md)